specification/level-1-version-5/sources/introduction.tex

```diff
@@ -0,0 +1,182 @@
+% ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+%% INTRODUCTION
+% ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+\chapter{Introduction}
+The Simulation Experiment Description Markup Language (SED-ML) is an XML-based format for describing simulation experiments, including model changes, calibrations, simulations, analyses, and computations and visualizations of simulation results.
+
+The number of computational models of biological systems is growing at an ever increasing pace.
+At the same time, their size and complexity are also increasing. It is now generally accepted that one must be able to exchange the mathematical structure of such models, for instance to build on existing studies by reusing models or to reproduce model results. The efforts to standardize the representation of computational models in various areas of biology, such as the Systems Biology Markup Language (SBML) \citep{Hucka:2003}, CellML \citep{cuellar:2003} or NeuroML \citep{Goddard:2001}, resulted in an increase of the exchange and re-use of models. However, the description of models is not sufficient to reproduce, reuse, and combine simulation experiments and their results. One also needs to describe the procedures the models are subjected to, i.e., the information required to reproduce simulation experiments among users and software tools. The increasing use of computational simulation experiments to inform modern biological research creates new challenges to reproduce, annotate, archive, and share such experiments.
+
+SED-ML is a computer-readable exchange format for describing simulation experiments. Critically, SED-ML can be used with a wide range of model formats, modeling frameworks, simulation algorithms, and simulation tools. For many simulation experiments, SED-ML can capture the information in the Minimum Information About a Simulation Experiment (MIASE) \citep{Waltemath:2011} guidelines.
+
+SED-ML is developed as a community project and defined via this technical specification and a corresponding XML Schema. 
+
+This document describes \currentLV of SED-ML, which is the successor of \previousLV and \LoneVone (described in \citep{WAB+11}).
+
+%-----------------------------------------------------------------------------
+\subsection{Color conventions in this document}
+\label{sec:notation}
+%-----------------------------------------------------------------------------
+
+Throughout this document, we use coloring to carry additional
+information for the benefit of those viewing the document on media
+that can display color:
+
+\begin{itemize}
+
+\item We use a \changed{red color} in text and a \textcolor[rgb]{0.0117, 0.0664, 0.5742}{dark blue color} in figures to indicate changes
+  between this version of the specification, namely SED-ML Level~1 Version~4, and
```

Comment:
should probably use the `\currentLV` and `\previousLV` macros to avoid these getting out of date...